irohad/torii/processor/transaction_processor.hpp

```diff
@@ -42,17 +42,16 @@ namespace iroha {
        * @param transaction - transaction for processing
        */
       virtual void transactionHandle(
-          std::shared_ptr<shared_model::interface::Transaction>
-              transaction) const = 0;
+          std::shared_ptr<shared_model::interface::Transaction> transaction)
+          const = 0;
 
       /**
        * Process transaction sequence and propagate batches from it either to
```

Comment:
"Process batch and propagate them either..." - after all, now we work with batches, not TxSequences